app/models/content_presenter.js

```diff
@@ -0,0 +1,101 @@
+var fs = require('fs');
+var Promise = require('bluebird');
+var readFile = Promise.promisify(fs.readFile);
+var glob = Promise.promisify(require('glob'));
+
+var RelatedContent = require('./related_content.js');
+var BreadcrumbMaker = require('../../lib/js/breadcrumb_maker.js');
+var TaxonomyData = require('./taxonomy_data.js');
+var Taxon = require('./taxon.js');
+
+class ContentPresenter {
+  constructor (basePath) {
+    this.basePath = basePath.slice(1, basePath.length); // base path without leading slash
+    this.baseName = this.basePath.replace(/\//g, '_');
+    this.contentDir = __dirname + "/../content"
+  }
+
+  present () {
+    const that = this;
+
+    return glob(that.contentDir + "/**/" + that.baseName + ".html").
+      then(function (file) {
+        const filePath = file[0];
+        const fileInfo = readFile(filePath).then(function (fileData) {
+          return {
+            fileData: fileData,
+            filePath: filePath
+          };
+        });
+
+        return fileInfo
+      }).
+      then(function (fileInfo) {
+        const content = fileInfo.fileData.toString();
+        const filePath = fileInfo.filePath;
+
+        var isHtmlPublication = content.match(/html-publications-show/);
+        var isWhitehall = filePath.match(/whitehall/);
+        var isHtmlManual = filePath.match(/manual/);
+
+        if (!isHtmlPublication) {
```

Comment:
I don't think we will have HTML publications for education. Should we delete the logic?